test/e2e/integration/frontend-test/pdf.ts

```diff
@@ -18,6 +18,28 @@ describe('PDF', () => {
     });
   });
 
+  it('downstream requests includes trace context header', () => {
+    const traceparent = '00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01';
+    const tracestate = 'altinn';
+    const cookies = {
+      'altinn-telemetry-traceparent': traceparent,
+      'altinn-telemetry-tracestate': tracestate,
+    };
+    for (const [key, value] of Object.entries(cookies)) {
+      cy.setCookie(key, value);
+    }
+    cy.goto('message');
+
+    cy.intercept({ method: 'GET', url: '**/applicationmetadata' }).as('appMetadata');
+
+    cy.testPdf('message', () => {
```

Comment:
The first argument here is the name of the (percy) snapshot to take (which will get `(PDF)` appended to it). There is already a PDF-test with this snapshot name, but in this case its probably not necessary to take a snapshot at all so you can set it to `false` instead.